src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -4199,6 +4260,7 @@ __meta__ = {
 }
 
 [node name="Report" type="Control" parent="MicrobeEditorGUI"]
+visible = false
```

Comment:
I think this and the other visible change, may be accidents? As these break the tab order.

Reply:
Yes, I forgot to change it back after testing